src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,362 @@
+        player2.setHandPlayer(result.getHand2());
+        firstDecisionSmallBlind = true;
+        playerFold = false;
+    }
+
+    public double amountToCall(Player player1, Player player2) {
+        return player1.getAmountInHand() - player2.getAmountInHand();
+    }
+
+    public void preFlopDetails(Player playerBigBlind, Player playerSmallBlind) {
+        System.out.println(result.getBoard());
+        System.out.println(playerBigBlind.getNamePlayer());
+        System.out.println(playerBigBlind.getHandPlayer());
+        System.out.println(playerBigBlind.getStackPlayer());
+        System.out.println(playerSmallBlind.getNamePlayer());
+        System.out.println(playerSmallBlind.getHandPlayer());
+        System.out.println(playerSmallBlind.getStackPlayer());
+        updatePotAndStackAmount(gameState.bigBlind, playerBigBlind);
+        updatePotAndStackAmount(gameState.smallBlind, playerSmallBlind);
+        playerSmallBlind.increasePlayerMoneyAtStrit(gameState.smallBlind);
+        playerBigBlind.increasePlayerMoneyAtStrit(gameState.bigBlind);
+        System.out.println("Zaczyna " + playerSmallBlind.getNamePlayer());
+        System.out.println(playerBigBlind.getNamePlayer() + " bigBlind " + gameState.bigBlind + "$");
+        System.out.println(playerSmallBlind.getNamePlayer() + " smallBlind " + gameState.smallBlind + "$");
+        printPot();
+        System.out.println(playerSmallBlind.getNamePlayer() + " podejmuje decyzje");
+        System.out.println(playerSmallBlind.getNamePlayer() + " " + amountToCall(playerBigBlind, playerSmallBlind) + "$ to call");
+        // System.out.println("1 = call 2 = raise");
+    }
+
+    public void printPot() {
+        System.out.println("Pot wynosi " + gameState.getPot() + "$");
+    }
+
+    public void layCards(GameState gameState) {
+        boolean isDraw = false;
+        if (gameState.getGamePhase() == GamePhase.PREFLOP) {
+            printPot();
+            System.out.println("Flop");
+            printBoard(gameState.getGamePhase());
+        } else if (gameState.getGamePhase() == GamePhase.FLOP) {
+            printPot();
+            System.out.println("Turn");
+            printBoard(gameState.getGamePhase());
+        } else if (gameState.getGamePhase() == GamePhase.TURN) {
+            printPot();
+            System.out.println("River");
+            printBoard(gameState.getGamePhase());
+        } else {
+            GameResult.result whoWins = equityEvaluator.updatePlayerWins(result.getBoard(), result.getHand1(), result.getHand2());
+            System.out.print(whoWins + " " + gameState.getPot() + "$");
+            if (whoWins == GameResult.result.DRAW) {
+                isDraw = true;
+            }
+            if (whoWins == GameResult.result.WIN_PLAYER1) {
+                updatePlayersOnShowdownResult(player1, player2, isDraw);
+            } else {
+                updatePlayersOnShowdownResult(player2, player1, isDraw);
+            }
+        }
```

Comment:
nie podoba mi sie ze to jest w funkcji layCards() jako ze funkcja wskazuje ze bedzie "Wykladala" karty a to jest dosc znaczacy kod ktory nie wpisuje sie w ta nazwe